mungegithub/submit-queue/ingress.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  name: k8s-submit-queue
+spec:
+  rules:
+  - host: kubernetes.submit-queue.k8s.io
+    http:
+      paths:
+        - backend:
+            serviceName: k8s-kubernetes-sq-status
+            servicePort: 80
+  - host: contrib.submit-queue.k8s.io
```

Comment:
Okay. But we wouldn't even be creating the ingress till the next PR is merged, so, I kept it in. I'll get rid of it.